violetplugin-communicationdiagram/src/main/java/com/horstmann/violet/product/diagram/communication/node/ActorNode.java

```diff
@@ -172,6 +176,20 @@ public void setName(SingleLineText newValue)
 //        getContent().refresh();
```

Comment:
This and other below are some comments I dont recall making - must have been cloned from the other repo, will fix as well.